amr-wind/equation_systems/icns/source_terms/DragForcing.cpp

```diff
@@ -0,0 +1,153 @@
+#include "amr-wind/equation_systems/icns/source_terms/DragForcing.H"
+#include "amr-wind/utilities/IOManager.H"
+
+#include "AMReX_Gpu.H"
+#include "AMReX_Random.H"
+#include "amr-wind/wind_energy/ABL.H"
+
+namespace amr_wind::pde::icns {
+
+DragForcing::DragForcing(const CFDSim& sim)
+    : m_sim(sim)
+    , m_mesh(sim.mesh())
+    , m_velocity(sim.repo().get_field("velocity"))
+{
+    amrex::ParmParse pp("DragForcing");
+    pp.query("dragCoefficient", m_drag);
+    pp.query("spongeStrength", m_spongeStrength);
```

Comment:
same remark about case here. I think we typically use snake case in the input files